components/scream/src/dynamics/homme/atmosphere_dynamics.hpp

```diff
@@ -47,9 +47,14 @@ class HommeDynamics : public AtmosphereProcess
   // Register all fields in the given repo
   void register_fields (FieldRepository<Real>& field_repo) const;
 
+  void set_required_group (const FieldGroup<const Real>& group);
```

Comment:
Worth explaining in a comment that "TRACERS TENDENCY" is the required group, and "TRACERS" is the updated group for dynamics?